model/Check/System/StuckTasksCheck.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace oat\taoSystemStatus\model\Check\System;
+
+use common_report_Report as Report;
+use oat\oatbox\log\LoggerAwareTrait;
+use oat\tao\model\taskQueue\TaskLog\Broker\TaskLogBrokerInterface;
+use oat\tao\model\taskQueue\TaskLog\TaskLogFilter;
+use oat\tao\model\taskQueue\TaskLogInterface;
+use oat\taoSystemStatus\model\Check\AbstractCheck;
+
+class StuckTasksCheck extends AbstractCheck
+{
+    use LoggerAwareTrait;
+
+    protected function doCheck(): Report
+    {
+        $message = '';
+        $allStuckTasks = $this->getAllStuckTasks();
+
+        if (empty($allStuckTasks)){
+            $report = new Report(Report::TYPE_SUCCESS, __('There is no stuck tasks'));
+        } else {
+            foreach ($allStuckTasks as $stuckTask) {
+                $task = [
+                    'task_id' => $stuckTask['task_id'],
```

Comment:
task_id and status are reusable strings and can be declared in constants
For example:
const OPTION_TASK_ID = 'task_id';

Reply:
done